aten/src/ATen/native/native_functions.yaml

```diff
@@ -4771,6 +4771,11 @@
 - func: sum.DimnameList_out(Tensor self, Dimname[1] dim, bool keepdim=False, *, ScalarType? dtype=None, Tensor(a!) out) -> Tensor(a!)
   device_check: NoCheck   # TensorIterator
 
+# TODO: this function will be replaced once nested expand semantics have been settled on
+- func: _nested_sum_backward(Tensor grad, Tensor self, int[1]? dim, bool keepdim=False) -> Tensor
+  dispatch:
+    NestedTensorCPU: NestedTensor_sum_dim_backward_CPU
```

Comment:
tiny nit: there isn't really any enforcement about naming conventions in the dispatch table, but maybe name the kernel `_nested_sum_backward_cpu`? (i.e. `{foo}_cpu` for the cpu kernel for an op `{foo}`.